resources/public/api/conf/2.0/examples/adjustments/foreign_property_accounting_adjustments_submit_response.json

```diff
@@ -0,0 +1,15 @@
+{
+  "id": "717f3a7a-db8e-11e9-8a34-2a2ae2dbcce4",
+  "links":[
+    {
+      "href":"/individuals/self-assessment/adjustable-summary/{nino}/foreign-property/{bsasId}/adjust",
+      "rel":"self",
+      "method":"GET"
+    },
+    {
+      "href":"/individuals/self-assessment/adjustable-summary/{nino}/foreign-property/{bsasId}",
+      "rel":"self",
+      "method":"GET"
+    }
+  ]
+}
```

Comment:
The example responses should contain the values for nino and bsasId

Also from the Hateoas links given in the glossary it is apprent that for the link for Retrieve BSAS (Foreign property) Should have a rel of `retrieve-adjustable-summary`

It should also have adjustedStatus set to true as a query parameter